app/models/user.rb

```diff
@@ -1,5 +1,13 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: Relationship.name,
+                                  foreign_key: "follower_id",
```

Comment:
```suggestion
                                  foreign_key: :follower_id,
```
fix dòng trên k fix dòng dưới à e